setup.py

```diff
@@ -2,45 +2,33 @@
 from setuptools import setup, find_packages
 
 
-setup_requires=[
+setup_requires = [
     'cfgov-setup==1.2',
     'setuptools-git-version==1.0.3',
 ]
 
 
 install_requires = [
     'Django>=1.8,<1.12',
-    'django-haystack==2.7.0',
+    'django-haystack==2.7.0',  # Latest version that supports both Django 1.8 and 1.11
     'djangorestframework==3.6.4',
-    'elasticsearch==2.4.1',
-    'PyYAML==3.11',
-    'python-dateutil==2.2',
-    'requests==2.7.0',
-    'Unipath==1.1',
+    'elasticsearch>=2.4.1,<3',
+    'PyYAML>=3.11,<3.14',
+    'python-dateutil>=2.2,<3',
```

Comment:
Super minor and annoying thing: in cfgov-refresh this is currently [`==2.1`](https://github.com/cfpb/cfgov-refresh/blob/138b03b5e400aa539409cd7b0700cf71390a7253/requirements/libraries.txt#L42). The changelog entry for 2.2 is [here](https://dateutil.readthedocs.io/en/stable/changelog.html#version-2-2). I don't really know enough about that library at first glance to know if those are big things we need to worry about or not. But:

It looks to me like the only use of `dateutil` in this whole repository is in [this unit test file](https://github.com/cfpb/college-costs/blob/a75769f0df9d1ceabaf991cf83e4dba8b48c308b/paying_for_college/tests/test_models.py#L351). So perhaps this could/should be moved out to `testing_extras`? If you look into that test file, it looks to me like that library could even be removed in favor of just creating the timestamps directly. So maybe we could even just strip it out?

Reply:
Well spotted. Does this work: f340146?

Reply:
I see, so the tests don't even use that timestamp at all. Nice fix.